federated-server/routes/supportRequests/create.js

```diff
@@ -11,9 +46,17 @@ export const createSupportRequest = async (req, res) => {
         const supportRequestData = orderLog(req.body);
         if (supportRequestData)
             sbot.add(supportRequestData,
-                async (err, supportRequest) => {
-                    supportRequest.value.content.status = await getStatus(sbot)(supportRequest.key);
-                    return res.json({ err, supportRequest });
+                async (error, supportRequest) => {
+                    if (error) return res.json({ error })
+                    try {
```

Comment:
We have planned to use a cron like job to hit the tier server with all pending support requests.
Should we change our mind?

Reply:
If we edit the status in ssb then we can make a query to reactivate those that could not be sent in some function of the scheduler.